vector/src/main/java/im/vector/app/core/di/ActiveSessionHolder.kt

```diff
@@ -80,18 +87,27 @@ class ActiveSessionHolder @Inject constructor(
     }
 
     fun hasActiveSession(): Boolean {
-        return activeSessionReference.get() != null
+        return activeSessionReference.get() != null || authenticationService.hasAuthenticatedSessions()
     }
 
     fun getSafeActiveSession(): Session? {
-        return activeSessionReference.get()
+        return runBlocking { getOrInitializeSession(startSync = true) }
```

Comment:
ideally `getActiveSession` and `getSafeActiveSession` would be suspending instead of using `runBlocking` however the change needed is much bigger than the scope of this PR